slybot/slybot/plugins/scrapely_annotations/processors.py

```diff
@@ -465,3 +478,10 @@ def __repr__(self):
         return u'%s(%s, field=%s, extractor=%s, adaptors=%s)' % (
             self.__class__.__name__, str(self), self._field, self.extractor,
             self.adaptors)
+
+
+class ProcessedItemField(ItemField):
```

Comment:
This can be accomplished with
```
class MetaField(ItemField):
    def _field_type(self):
       return META_FIELD
```
This would be the first step to break apart the `ItemField` class into the `DefaultField`, `ProcessingField` and `MetaField` subclasses.